frontend/src/apis/index.js

```diff
@@ -0,0 +1,39 @@
+// Is an abstraction to communicate with backend
+
+class API {
+    BASE_URL = '/api';
+    myFetch = (url, config) =>
+        window.fetch(this.BASE_URL + url, {
+            headers: { 'content-type': 'application/json' },
+            ...config,
+        });
+
+    async login({ pin, phoneNumber }) {
+        try {
+            return await this.myFetch(`/users/login`, {
+                method: 'POST',
+                body: JSON.stringify({
```

Comment:
The body will always be JSON and thus should be formatted in the fetch method to keep it simple to add new methods here.

Reply:
good idea